include/cutlass/pipeline/sm90_pipeline.hpp

```diff
@@ -417,7 +417,7 @@ private :
     }
 
     // Most likely you have elected more than one leader
-    if (params_.is_leader && (threadIdx.x % 32 != 0)) {
+    if (params_.is_leader && (ThreadIdxX() % 32 != 0)) {
```

Comment:
This came up as an error when building the pipeline with `-DCMAKE_BUILD_TYPE=Debug` flag.